R/plot_cbass.R

```diff
@@ -904,6 +904,9 @@ cbass_path_plotly <- function(x,
   plot_frame_full <- plot_frame_full %>% filter(.data$GammaPercent <= max(percent, min(.data$GammaPercent)))
 
   plot_frame_init  <- plot_frame_full %>% filter(.data$Iter == min(.data$Iter))
+  plot_frame_init_adj <- get_ggrepel_coordinates(plot_frame_init$V1, plot_frame_init$V2, plot_frame_init$ObsLabel)
+  plot_frame_init_dist <- plot_frame_init_adj %>% mutate(dist = sqrt((x-x_adj)^2+(y-y_adj)^2))
```

Comment:
This needs `.data` around `x`, `x_adj`, `y`, and `y_adj` to prevent check warnings.